src/Confluent.Kafka/Admin/DeleteAclsReport.cs

```diff
@@ -0,0 +1,111 @@
+// Copyright 2022 Confluent Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Refer to LICENSE for more information.
+
+using System;
+using System.Linq;
+using System.Collections.Generic;
+
+
+namespace Confluent.Kafka.Admin
+{
+    /// <summary>
+    ///    Report for a delete ACLs operation with an <see cref="Error" /> and a list of <see cref="AclBinding" />
+    /// </summary>
+    public class DeleteAclsReport
+    {
+        /// <summary>
+        ///     Result of this report
+        /// </summary>
+        public DeleteAclsResult Result { get; set; }
```

Comment:
I think it's better if you don't use DeleteAclsResult here and replicate the property instead. Regardless, this is the convention set by DeleteRecords, so good to follow it.

Reply:
Done